include/maths/CBoostedTree.h

```diff
@@ -113,6 +117,9 @@ namespace boosted_tree {
 
 //! \brief Computes the leaf value which minimizes the loss function.
 class MATHS_EXPORT CArgMinLoss {
+public:
+    using TDouble1Vec = core::CSmallVector<double, 1>;
```

Comment:
Calling it `TDouble1Vec` implies that the vector has only one element. How would you extend it to the multinomial case, introducing `TDoubleNVec`?

Reply:
This is just the convention we use throughout the code base to distinguish from `std::vector`. We use the `N` in `TDouble<N>Vec` to denote the number of values the vector can store before it starts using the heap. For consistency I'd prefer to leave this as is.